lib/index.js

```diff
@@ -372,6 +373,8 @@ class Avanza extends EventEmitter {
   * @param {Object} credentials
   * @param {String} credentials.username
   * @param {String} credentials.password
+  * @param {String} credentials.totp
+  * @param {String} credentials.totp_secret
```

Comment:
Perhaps we should rename this to `totpSecret`, just to follow camel case convention.